api/test/wfl/unit/modules/xx_test.clj

```diff
@@ -9,17 +9,17 @@
         inputs (xx/make-inputs-to-save output-url {:input_cram sample})]
     (is (= sample (:input_cram inputs)))
     (is (= "sample" (:sample_name inputs)))
-    (is (= "sample.cram" (:base_file_name inputs)))
-    (is (= "sample.cram" (:final_gvcf_base_name inputs)))
+    (is (= "sample" (:base_file_name inputs)))
+    (is (= "sample" (:final_gvcf_base_name inputs)))
```

Comment:
The test agrees with the example inputs now.

https://github.com/broadinstitute/warp/blob/e64b2499b1a04e3df701556231c317200cf30ee1/pipelines/broad/reprocessing/external/exome/ExternalExomeReprocessing.plumbing.input.json#L5-L6